src/host/outputStream.cpp

```diff
@@ -256,23 +256,23 @@ BOOL ConhostInternalGetSet::PrivateGetExtendedTextAttributes(ExtendedAttributes*
 // Arguments:
 // - extendedAttrs: The new ExtendedAttributes to use
 // Return Value:
-// - TRUE if successful (see DoSrvPrivateSetExtendedTextAttributes). FALSE otherwise.
-BOOL ConhostInternalGetSet::PrivateSetExtendedTextAttributes(const ExtendedAttributes attrs)
+// - true if successful (see DoSrvPrivateSetExtendedTextAttributes). false otherwise.
+bool ConhostInternalGetSet::PrivateSetExtendedTextAttributes(const ExtendedAttributes& attrs)

```

Comment:
I think ExtAttr is a value type; any reason you chose a ref?

Reply:
I thought it was a full class. Turning it back.

Reply:
(mostly because the one below it which is very similar is a whole class)